example/flask_op/config_martin.yaml

```diff
@@ -0,0 +1,309 @@
+logging:
```

Comment:
is it needed? :)

Reply:
Not really.

Reply:
The config_martin.yaml file is removed. Not updated and found no compelling reason to do it. Removed it instead.